### ISSUE TITLE
Stasis dermatitis

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6576]
**Preferred term label:**  Stasis dermatitis

**Synonyms**
Venostasis

**Definition (free text, please give PubMed ID)**
An inflammatory process of the skin resulting from venous hypertension resulting from retrograde flow due to incompetent venous valves, valve destruction, or obstruction of the venous system PMID: 28063094

**Parent term (use hpo.jax.org/app)**
Abnormality of venous physiology HP:0030846

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
adding, but as a child of eczema.
